pytest_reportportal/listener.py

```diff
@@ -0,0 +1,45 @@
+import html
```

Comment:
Does `html` module exist in the `Python2.7`? Also this module is no a dependency of `pytest`. Than is why solution with `cgi` was chosen.

Reply:
You are right. Fixed.